modules/installation-uninstall-clouds.adoc

```diff
@@ -1,28 +1,23 @@
 // Module included in the following assemblies:
 //
 // * installing/installing_aws/uninstalling-cluster-aws.adoc
+// * installing/installing_azure/uninstalling-cluster-azure.adoc
+// * installing/installing_gcp/uninstalling-cluster-gcp.adoc
 
-[id="installation-uninstall-aws_{context}"]
-= Removing a cluster from AWS
+[id="installation-uninstall-clouds_{context}"]
+= Removing a cluster that uses installer-provisioned infrastructure
 
-You can remove a cluster that you installed on Amazon Web Services (AWS).
+You can remove a cluster that uses installer-provisioned infrastructure from
+your cloud.
 
 .Prerequisites
 
 * Have a copy of the installation program that you used to deploy the cluster.
+* Have the files that the installation program generated when you created your
+cluster.
```

Comment:
@kalexand-rh Will the user know what/where these files are?

Reply:
@mburke, they should. I have a note in the installation files that says "You must not delete the installation program or the files that the installation program creates. Both are required to delete the cluster."